docs/javascript/v2/features/simulcast.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Simulcast
+nav: 3.33
+---
+
+To turn on simulcast in your room, open the [templates](https://dashboard.100ms.live/templates) in the dashboard and enable it for roles there.
```

Comment:
a screenshot would be helpful

Reply:
not on prod. deployed to qa recently I think. was waiting for that. will update.